<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_line">
       <property name="text">
        <string>线性变换</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>斜率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_k"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>截距</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_b"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pb_log">
       <property name="text">
        <string>对数变换</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>lambda乘积</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_lamb_log"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>base对数底</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_base_log"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pb_power">
       <property name="text">
        <string>指数变换</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>lambda乘积</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_lamb_power"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>指数幂</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_power"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pb_hisEqual">
       <property name="text">
        <string>直方均衡</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_lut">
       <property name="text">
        <string>直方规定(默认映射表)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
